applications/luci-app-wireguard/luasrc/view/wireguard.htm

```diff
@@ -169,100 +79,117 @@
 
   XHR.poll(5, '<%=REQUEST_URI%>', { status: 1 },
    function(x, data) {
-    for (var i = 0, ilen = data.length; i < ilen; i++) {
-      var iface = data[i];
-      var ifid = iface['public_key'] + "_";
-      var s = String.format(
-        '<strong><%:Public Key%>: </strong>%s' +
-        '<br /><strong><%:Listening Port%>: </strong>%<%:s%>',
-        iface['public_key'],
-        iface['listening_port']
-      );
-      document.getElementById(ifid + "info").innerHTML = s;
-      for (var j = 0, jlen = iface['peers'].length; j < jlen; j++) {
-        var peer = iface['peers'][j];
-        var pid = ifid + peer['public_key'] + "_";
-        s = String.format(
+    for (var key in data) {
+      if (!data.hasOwnProperty(key)) { continue; }
+      var ifname = key;
+      var iface = data[key];
+      var s = "";
+      if (iface.public_key == '(none)') {
+        s += '<em><%:Interface not operational!%></em>';
```

Comment:
public-key <-- This now displays a "Interface/Peer not operational" message to indicate that it is not a GUI problem that nothing shows up. What do you think?